corehq/apps/data_dictionary/util.py

```diff
@@ -229,3 +229,13 @@ def fields_to_validate(domain, case_type_name):
     }
     props = CaseProperty.objects.filter(**filter_kwargs)
     return {prop.name: prop for prop in props}
+
+
+@quickcache(['domain', 'case_type'], timeout=24 * 60 * 60)
+def get_smart_types_by_prop(domain, case_type):
+    # Used for CASE_SEARCH_SMART_TYPES
+    return dict(CaseProperty.objects.filter(
+        case_type__domain=domain,
+        case_type__name=case_type,
+        data_type='gps',
```

Comment:
nit: might be worth making this a constant, since you reference it in many places.

Reply:
I think this would still be good to do.

Reply:
:+1: